### PR TITLE
fix: fix block preview image wrong DisplayMode

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Fixes bug "Bug: The block image preview isn’t generated correctly"

## How Has This Been Tested?
Manual

## Documentation
None

### CODE REVIEW

File: packages/app-page-builder-editor/src/components/Editor/Editor.scss [hunk 1]
```diff
@@ -54,5 +54,8 @@
     .webiny-pb-page-element-container::after {
       content: none;
     }
+    .webiny-pb-base-page-element-style::after {
```
Comment (neatbyte-vnobis) [inline packages/app-page-builder-editor/src/components/Editor/Editor.scss]: @Pavel910 it should be copied to both `app-page-builder-editor` and `app-page-builder` packages, right?

Comment (Pavel910) [reply packages/app-page-builder-editor/src/components/Editor/Editor.scss]: @neatbyte-vnobis You can completely ignore `app-page-builder-editor` package. It is set to `private`, so it's not being published. It's my sandbox for some experiments.

File: packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx [hunk 2]
```diff
@@ -9,6 +9,9 @@ import { EditorBar } from "~/editor";
 import { useEventActionHandler } from "~/editor/hooks/useEventActionHandler";
 import { useBlock } from "~/blockEditor/hooks/useBlock";
 import { SaveBlockActionEvent } from "~/blockEditor/config/eventActions/saveBlock/event";
+import { useUI } from "~/editor/hooks/useUI";
+import { setDisplayModeMutation } from "~/editor/recoil/modules";
+import { DisplayMode } from "~/types";
```
Comment (Pavel910) [inline packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx]: @neatbyte-vnobis let's please hide this low-level code inside of the [useDisplayMode](https://github.com/webiny/webiny-js/blob/990c3ab1b6577fb6f438c5ba0beacd6435e96f2d/packages/app-page-builder/src/editor/hooks/useDisplayMode.tsx#L12) hook. Just add a `setDisplayMode` function and hide the internals there in one place.

File: packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx [hunk 3]
```diff
@@ -20,19 +23,23 @@ const DefaultSaveBlockButton: React.FC = () => {
     const { history } = useRouter();
     const { showSnackbar } = useSnackbar();
     const [loading, setLoading] = useState(false);
+    const [, setUiValue] = useUI();
```
Comment (Pavel910) [inline packages/app-page-builder/src/blockEditor/config/editorBar/SaveBlockButton/SaveBlockButton.tsx]: Once you have the new method in the `useDisplayMode` hook, use it here.

**Review by Pavel910 — CHANGES_REQUESTED** [review]
Let's improve the implementation and have the `setDisplayMode` function in the `useDisplayMode` hook.

**Review by Pavel910 — APPROVED** [review]

**Pull request merged**